docs/content/Coding Milestones/PoC2023q1/outline.md

```diff
@@ -153,8 +153,8 @@ in an eventually consistent way, it is not just one-and-done.
 
 In this design the primary interface between infrastructure management
 and workload management is API objects in _inventory management_
-workspaces.  We abuse the `Location` and `SyncTarget` object types
-from [kcp TMC](https://github.com/kcp-dev/kcp/tree/v0.11.0/pkg/apis) for
+workspaces.  We use the `Location` and `SyncTarget` object types
+from Kubestellar's edge.kcp.io api group (created by `kubestellar init`) for
```

Comment:
The api group is not "created" by `kubestellar init`.
Those object types are not "created" by `kubestellar init`.
The `kubestellar init` command makes the KubeStellar Core Space export those resources.

Reply:
ok, yes, make sense